### ISSUE TITLE
Limit of tests to be retried

### ISSUE THREAD

**Issue by IlyaHarlamov** [prashant-ramcharan/courgette-jvm#336]
Hello, is it possible to limit the amount of tests to be retried or to limit amount of total retries per execution?

**Comment by IlyaHarlamov** [comment]
Hi, great, thank you for the quick reply! I have discovered your project just recently but I really like the concept, there is one more idea for the future, it might be also a good idea to implement an option to exclude the tests from the rerun by a cucumber tag. So far I only came up with 2 workarounds, but both of then are not looking clean:
1. Separating these tests in a single feature file and using `excludeFeatureFromRerun`
2. Split execution in 2 runners (or execute a single runner 2 times with different parameters), where first runner is going to execute the tests with enabled rerun feature, by excluding tags e.g. `cucumber.tags="not @no-rerun"` and second runner with disabled rerun feature, where only non-rerun tests will be executed.

**Comment by prashant-ramcharan** [comment]
I think adding a new Courgette option called `excludeTagFromRerun` should be fairly straight forward to add. I will try to include this into the next release.

**Comment by prashant-ramcharan** [comment]
This is now released in Courgette `6.5.1`.

Like other Courgette options, you can define the values in the Courgette runner or pass it at runtime

````java
@RunWith(Courgette.class)
@CourgetteOptions(
     excludeTagFromRerun = {"@tag1", "@tag2"}
))
public class CourgetteRunner {
}
````

_or_

````
-Dcourgette.excludeTagFromRerun="@tag1, @tag2"
````
